### DISCUSSION TITLE
Does browsergym support som observation now?

### DISCUSSION THREAD

**Discussion by junleiz** [ServiceNow/BrowserGym#115] [General]

**Comment by gasse** [comment]
Yes, browsergym supports som. You can use this method to add boxes to the screenshot:
https://github.com/ServiceNow/BrowserGym/blob/87c45ad46a9f7b992f286383bf04c54c53c9882e/browsergym/core/src/browsergym/utils/obs.py#L425

Example here:
https://github.com/ServiceNow/AgentLab/blob/f385533ce85b89ad9456f676ebd019221d968f85/src/agentlab/agents/dynamic_prompting.py#L819
